app/packs/stylesheets/decidim/metadecidim-theme/_navbar.scss

```diff
@@ -0,0 +1,5 @@
+@media print, screen and (min-width: 40em) {
```

Comment:
Just a stylistic suggestion but I think you could also use the foundation helpers here for the breakpoint definition:
```scss
.main-nav__link a {
  @include breakpoint(medium){
    padding: .75em 1em;
  }
}
```

Reply:
Yes, is much cleaner and prettier for sure. Done in 23b1aae5460ee45bc8e0f4f4acbf82b236f24b33